actors/builtin/power/policy.go

```diff
@@ -9,5 +9,4 @@ const ConsensusMinerMinMiners = 4 // PARAM_SPEC
 //
 // This limits the number of proof partitions we may need to load in the cron call path.
 // Onboarding 1EiB/year requires at least 32 prove-commits per epoch.
-const MaxMinerProveCommitsPerEpoch = 200 // PARAM_SPEC
-
+const MaxMinerProveCommitsPerEpoch = 1000 // PARAM_SPEC
```

Comment:
This needs a version guard too.

Will this only apply to the old non-batch proof? I hope we're planning to deprecate that, in which case this will probably never be hit and the effort to parameterise it not worthwhile.

Edit: oh is this just for testing? Hmm, maybe you need to spread the proofs out over a few epochs (which is realistic anyway.

Reply:
Yeah this was just for testing that doesn't need to be merged.